themes/spichka/resources/styles/post-card.scss

```diff
@@ -0,0 +1,36 @@
+// Post card
+
+.post-card,
+.post-card-extended {
```

Comment:
Instead of .post-card-extended you could .post-card{ &.extended { ... } }

Reply:
Maybe we will remove multiple cards, so I will leave it for now as is.